src/map/packets.h

```diff
@@ -3731,6 +3731,23 @@ packet(0x96e,-1,clif->ackmergeitems);
 	packet(0x0923,36,clif->pStoragePassword,0);
 	packet(0x0a27,8); // ZC_RECOVERY2
 	packet(0x09f7,75); // ZC_PROPERTY_HOMUN_2
+	packet(0x09E8,11,clif->rodex_open_mailbox); // CZ_OPEN_MAILBOX
```

Comment:
Thanks for the reference, but that's weird, 0x0a08 is on 2014-04 client but it doesn't have RoDEX and its length is really different (8 in this date compared to 26 in newer ones), what should I do in these cases?